### PR TITLE
Code of conduct

### PR DESCRIPTION
**Pull request by graeme-a-stewart**
Replacing #68 with the branch hosted in the main repository

### CODE REVIEW

File: content/network/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+* 2018-09-11 Code of Conduct revised to provide straightforward examples of both beneficial and unwanted behaviour. Also includes changes regarding evaluating intent. Contributors: Ethan White, Kari L. Jordan, Karin Lagesen, Malvika Sharan, Samantha Ahern, and Simon Waldman.
+* 2018-09-11 Reporting guidelines revised to include alternate contact points. Contributors: Ethan White, Kari L. Jordan, Karin Lagesen, Malvika Sharan, Samantha Ahern, and Simon Waldman.
+* 2016-11-21 Code of Conduct, Reporting Guidelines and Enforcement Manual released. Documents were adapted from guidelines written by the Django Project, which was itself based on the Ada Initiative template and the PyCon 2013 Procedure for Handling Harassment Incidents. Contributors: Adam Obeng, Aleksandra Pawlik, Bill Mills, Carol Willing, Erin Becker, Hilmar Lapp, Kara Woo, Karin Lagesen, Pauline Barmby, Sheila Miguez, Simon Waldman, Tracy Teal.
+
+## About this Document
```
Comment (mihai-sysbio) [inline content/network/code_of_conduct.md:94]: A link to The Carpentries code of conduct would be nice.

File: content/network/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+Everyone who participates in EVERSE activities is required to conform to this Code of Conduct. It applies to all spaces managed by EVERSE including, but not limited to, events, workshops, meetings, email lists, and online forums such as GitHub, Slack, Mastodon,  X and LinkedIn. Event, workshop and meeting hosts are expected to assist with the enforcement of the Code of Conduct. By participating, participants indicate their acceptance of the procedures by which EVERSE resolves any Code of Conduct incidents, which may include storage and processing of their personal information.
```
Comment (mihai-sysbio) [inline content/network/code_of_conduct.md:41]:
```suggestion
Everyone who participates in EVERSE activities is required to conform to this Code of Conduct. It applies to all spaces managed by EVERSE including, but not limited to, events, workshops, meetings, email lists, and online forums such as GitHub, Slack, Mastodon, X, Bluesky and LinkedIn. Event, workshop and meeting hosts are expected to assist with the enforcement of the Code of Conduct. By participating, participants indicate their acceptance of the procedures by which EVERSE resolves any Code of Conduct incidents, which may include storage and processing of their personal information.
```

File: content/network/code_of_conduct.md (hunk 1 above)
Comment (SanjeAntona) [reply content/network/code_of_conduct.md]: done

File: content/network/code_of_conduct.md (hunk 2 above)
Comment (SanjeAntona) [reply content/network/code_of_conduct.md]: added and created Bluesky account

**Review by mihai-sysbio — APPROVED** [review]
Look good to me (note: have not deployed this)

**Pull request merged**